NDTensors/ext/NDTensorsCUDAExt/typeparameteraccessors.jl

```diff
@@ -0,0 +1,20 @@
+# # `TypeParameterAccessors.jl` overloads.
+using NDTensors.TypeParameterAccessors: TypeParameterAccessors, Position, parameter
+## TODO this seems like a `GPUArrays` generic function
+storagemode(t::Type{<:CuArray}) = parameter(t, 3)
+function TypeParameterAccessors.position(::Type{<:CuArray}, ::typeof(storagemode))
+  return Position(3)
+end
```

Comment:
Reminder to maybe add inverse definition:
```julia
function TypeParameterAccessors.parameter_function(::Type{<:CuArray}, ::Position(3))
  return storagemode
end
```

Reply:
Okay I have implemented the inverse definitions via `position_name`